XenAdmin/TabPages/NICPage.cs

```diff
@@ -163,6 +163,9 @@ private void updateList()
 
                         RegisterPIFEventHandlers(PIF);
 
+                        if (PIF.IsSriovLogicalPIF())

```

Comment:
Shouldn't this move above the event registration? I wouldn't think we need to register events for skipped PIFs.